xbmc/video/VideoUtils.cpp

```diff
@@ -635,7 +638,7 @@ ResumeInformation GetNonFolderItemResumeInformation(const CFileItem& item)
     db.GetResumeBookMark(path, bookmark);
     db.Close();
 
-    if (bookmark.IsSet())
+    if (bookmark.timeInSeconds > 0.0)
```

Comment:
Is this intentional? https://github.com/xbmc/xbmc/blob/82f7b7bd86a56c5aec6d28b44cb5133027b99b04/xbmc/video/Bookmark.cpp#L26
Seems to be the same

Reply:
> Is this intentional? 

 Yes.
 
> Seems to be the same

 No. `IsSet` checks for `totalTimeInSeconds > 0.0`. What I'm interested in is whether there is actually a resume time (`timeInSeconds > 0.0`).

Reply:
Just spotted that there already is `CBookMark::IsPartWay()` which fits here. Will change.